src/scripts/main.js

```diff
@@ -2,7 +2,34 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tbl = document.getElementsByClassName('dashboard')[0];
 
-// write your code here
+people.forEach(el => {
+  const tr = document.createElement('tr');
+  let td = document.createElement('td');
+
+  td.innerHTML = el.name;
+  tr.append(td);
+
+  td = document.createElement('td');
+  td.innerHTML = el.sex === 'f' ? 'Female' : 'Male';
+  tr.append(td);
+
+  td = document.createElement('td');
+  td.innerHTML = el.born;
+  tr.append(td);
+
+  td = document.createElement('td');
+  td.innerHTML = el.died;
+  tr.append(td);
+
+  td = document.createElement('td');
+  td.innerHTML = el.died - el.born;
+  tr.append(td);
+
+  td = document.createElement('td');
+  td.innerHTML = Math.ceil(el.died / 100);
+  tr.append(td);
+
+  tbl.append(tr);
```

Comment:
let's use `insertAdjacentHTML()` instead of repetitions. it will be much clearer and simpler

Reply:
done